### PR TITLE
fix ci

### PR DESCRIPTION
**Pull request by EnKarin**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@ jobs:
                 key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
                 restore-keys: ${{ runner.os }}-m2
           -   name: Build with Maven
+              run: mvn clean install
```
Comment (mfvanek) [inline .github/workflows/build.yml:35]: What does it do?
I mean what's problem you're trying to solve with this change?

**Review by mfvanek — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (EnKarin) [reply .github/workflows/build.yml:35]: The problem of building pr without access to secrets

Comment (mfvanek) [reply .github/workflows/build.yml:35]: It's enough here `mvn clean verify`.
You shouldn't do install command in such cases. 

And the next step should be replaced with `mvn verify -P sonar` - without **clean**

Comment (EnKarin) [reply .github/workflows/build.yml:35]: (it looks like a package is needed)

**Review by mfvanek — APPROVED** [review]

**Pull request merged**